### DISCUSSION TITLE
Problem when converting Apk to Aab

### DISCUSSION THREAD

**Discussion by studiobummbum** [iBotPeaches/Apktool#3514] [Q&A]
I don't know if the problem is with Apktool or with the apk to aab conversion. When I use the application in the Apk version, it is completely clean. When I convert to aab, when installing aab into Firebase testlab, it will report a missing file in the raw/... folder, leading to an app crash. Please help

**Comment by iBotPeaches** [comment]
Since you used the apk version w/ no issue. You've eliminated any possibility that Apktool is involved. Apktool at present cannot work with AABs.
